rust/automerge/src/columnar/encoding/col_error.rs

```diff
@@ -6,6 +6,14 @@ pub(crate) struct DecodeColumnError {
 
 impl std::error::Error for DecodeColumnError {}
 
+impl From<crate::storage::parse::leb128::Error> for DecodeColErrorKind {
```

Comment:
I don't think we need this `From` implementation right? Removing this also allows us to remove the `pub(crate)` on `DecodeColErrorKind` as well.

Reply:
Thanks! I was doing a lot of trial-and-error to get it working.

Pushed a new version now I've figured out why lifetimes were not working out for me before.